### PR TITLE
- Updated autoinstall.sh script to support the new `jq` utility added…

### PR DESCRIPTION
**Pull request by AuRoN89**
… in `FMods Tools 2.8`.

- Added uninstall.sh script to manage /images data uninstall.

### CODE REVIEW

File: SyncMyMod/autoinstall.sh [hunk 1]
```diff
@@ -25,102 +25,92 @@
...
-function dump_logs {
-	cp ${LOG_FILE} ${LOG_DIR}/install_status.txt
+dump_logs() {
+	cp "${LOG_FILE}" "${LOG_DIR}"/install_status.txt
 	sync
 }
 
-function output {
-	echo "${1}" > $DISPLAY
-	echo "${1}" >> $LOG_FILE
```
Comment (bigunclemax) [inline SyncMyMod/autoinstall.sh:95]: Is it okay that we drop the log output here?

File: SyncMyMod/autoinstall.sh [hunk 2]
```diff
@@ -25,102 +25,92 @@
...
 }
 
-function progress {
-	echo "PROGRESS ${1}" > $DISPLAY
+progress() {
+    echo "PROGRESS ${1}" > "$DISPLAY"
 }
 
-function displayMessage {
-	dump_logs
```
Comment (bigunclemax) [inline SyncMyMod/autoinstall.sh:104]: And here...

File: SyncMyMod/autoinstall.sh [hunk 3]
```diff
@@ -143,32 +133,36 @@ function collectHwInfo {
 ###############################################################################
 # Start mod installation                                                      #
 ###############################################################################
-echo "Starting installation of ${FANCYNAME} v${VERSION}..." > $LOG_FILE
 
+echo "Starting installation of ${FANCYNAME} v${VERSION}..." > "$LOG_FILE"
```
Comment (bigunclemax) [inline SyncMyMod/autoinstall.sh]: FANCYNAME was replaced with APP_NAME, isn't?

File: SyncMyMod/autoinstall.sh (hunk 2 above)
Comment (AuRoN89) [reply SyncMyMod/autoinstall.sh:104]: i recreated the functions with ()

File: SyncMyMod/autoinstall.sh (hunk 1 above)
Comment (AuRoN89) [reply SyncMyMod/autoinstall.sh:95]: i recreated the functions with ()

File: SyncMyMod/autoinstall.sh (hunk 3 above)
Comment (AuRoN89) [reply SyncMyMod/autoinstall.sh]: whops, yes, i forgot to update that line

**Pull request closed without merging**